quantum/split_common/transactions.c

```diff
@@ -578,6 +578,41 @@ static void st7565_handlers_slave(matrix_row_t master_matrix[], matrix_row_t sla
 
 #endif  // defined(ST7565_ENABLE) && defined(SPLIT_ST7565_ENABLE)
 
+////////////////////////////////////////////////////
+// POINTING
+
+#if defined(POINTING_DEVICE_ENABLE) && defined(SPLIT_POINTING_ENABLE)
+
+extern const pointing_device_driver_t pointing_device_driver;
+
+report_mouse_t get_targets_pointing(void) {
+    report_mouse_t temp = {0};
+
+    // Prevent transaction attempts while transport is disconnected
+    if (!is_transport_connected()) {
+        return temp;
+    }
+
+    if (!transport_execute_transaction(GET_POINTING, NULL, 0, &temp, sizeof(report_mouse_t))) {
+        dprintf("Failed to execute pointing devices transaction.");
+    }
+    return temp;
+}
+
+static void pointing_handlers_slave(uint8_t initiator2target_buffer_size, const void *initiator2target_buffer, uint8_t target2initiator_buffer_size, void *target2initiator_buffer) {
+    report_mouse_t temp = {0};
+    temp                = pointing_device_driver.get_report(temp);
```

Comment:
Just a heads up, this gets the raw report, without any processing from the kb code. 

It may be better to use `pointing_device_get_report()`, as this should be processed code.   Otherwise, "combined" report may not be able to separate the reports for each half.

Reply:
The idea was to grab the raw report and process it when required on the master side. I've not fully decided on how to do that yet. I should have the ability to test the combined stuff a little better within the new few days.

Reply:
Yeah, grabbing the raw data is probably the most correct option.  But definitely should be something there to "process" it, so it can be moved/handled different, if needed.